src/lib.rs

```diff
@@ -65,6 +81,10 @@ module! {
         pub struct GetLogs as "eth_getLogs"
             (LogFilter,) => Vec<Log>;
 
+        /// Returns the value from a storage position at a given address.
+        pub struct GetStorageAt as "eth_getStorageAt"
+            (Address, U256, Option<BlockSpec>) => Vec<u8> [serialization::bytes];
```

Comment:
I think this is incorrect. It will always return a 32-byte word if I’m not mistaken:

```suggestion
            (Address, U256, Option<BlockSpec>) => [u8; 32] [serialization::bytes];
```

You might need to adapt the `serialization::bytes` code to make this work. If you do, then I would take this particular change out of this PR and do it in a follow up.

Reply:
Sounds good! Suggestion applied.